pycbc/bank/conversions.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (C) 2022 Gareth Cabourn Davies
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+
+#
+# =============================================================================
+#
+#                                   Preamble
+#
+# =============================================================================
+#
+"""
+This module is supplied to make a convenience function for converting into
+specific values from PyCBC template banks.
+"""
+
+from pycbc import conversions as conv
+from pycbc import pnutils
+
+# Convert from parameter name to helper function
+# some multiple names are used for the same function
+conversion_options = ['mass1', 'mass2', 'spin1z', 'spin2z', 'duration',
+                      'template_duration', 'mtotal', 'total_mass',
+                      'q', 'invq', 'eta', 'chirp_mass', 'mchirp',
+                      'chieff', 'chi_eff', 'effective_spin', 'chi_a']
+
+
+mass_conversions = {
+    'mtotal': conv.mtotal_from_mass1_mass2,
+    'total_mass': conv.mtotal_from_mass1_mass2,
+    'q': conv.q_from_mass1_mass2,
+    'invq': conv.invq_from_mass1_mass2,
+    'eta': conv.eta_from_mass1_mass2,
+    'mchirp': conv.mchirp_from_mass1_mass2,
+    'chirp_mass': conv.mchirp_from_mass1_mass2,
+}
+
+spin_conversions = {
+    'chieff': conv.chi_eff,
+    'chi_eff': conv.chi_eff,
+    'effective_spin': conv.chi_eff,
+    'chi_a': conv.chi_a
+}
+
+
+def bank_conversion(parameter, bank, template_ids):
```

Comment:
Think I'd prefer to call this something less similar to the name of the package and module. Eg `def get_bank_property()` ?